cygnus-ngsi/pom.xml

```diff
@@ -58,7 +58,14 @@
             <artifactId>cygnus-common</artifactId>
             <version>2.9.0</version>
         </dependency>
-        
+        <!-- https://mvnrepository.com/artifact/org.jodd/jodd-util -->
+        <dependency>
+            <groupId>org.jodd</groupId>
+            <artifactId>jodd-core</artifactId>
+            <version>3.6.8</version>
+        </dependency>
```

Comment:
Where this new dependency is used? Java code modifications included in this PR are only deletions...

Reply:
Used by https://github.com/telefonicaid/fiware-cygnus/blob/master/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIArcgisFeatureTableSink.java#L58

Probably imported by hive-exec dep

Reply:
NTC